internal/pkg/config/config_test.go

```diff
@@ -49,10 +50,17 @@ func runTest(pass *analysis.Pass) (interface{}, error) {
 		}
 		for _, b := range f.Blocks {
 			for _, i := range b.Instrs {
-				if c, ok := i.(*ssa.Call); ok {
+				switch c := i.(type) {
+
+				case *ssa.Call:
 					if callee := c.Call.StaticCallee(); callee != nil && conf.IsSink(utils.DecomposeFunction(callee)) {
 						pass.Reportf(i.Pos(), "sink call")
 					}
+
+				case *ssa.Panic:
+					if !conf.AllowPanicOnTaintedValues {
```

Comment:
This doesn't test the behavior in `levee`. These tests should be in the `internal/pkg/levee` directory.

(This also doesn't test any config-specific behavior. I don't think we need to test this change here).

Reply:
I was thinking the same thing. I have moved the test to internal/pkg/levee directory.